src/UiPath.Workflow.Runtime/Runtime/CompletionBookmark.cs

```diff
@@ -19,19 +19,19 @@ public CompletionBookmark(CompletionCallbackWrapper callbackWrapper)
     }
 
     [DataMember(EmitDefaultValue = false, Name = "callbackWrapper")]
-    internal CompletionCallbackWrapper SerializedCallbackWrapper
+    public CompletionCallbackWrapper SerializedCallbackWrapper
```

Comment:
changes to this file are no longer needed

Reply:
Fixed.